cardano-testnet/test/Spec/Plutus/Direct/TxInLockingPlutus.hs

```diff
@@ -0,0 +1,238 @@
+  utxoSKeyFile <- H.note $ tempAbsPath </> "shelley/utxo-keys/utxo1.skey"
+
+  -- ("CARDANO_NODE_SOCKET_PATH", IO.sprocketArgumentName (head bftSprockets))
+
+  plutusScriptFileInUse <- H.note $ base </> "scripts/plutus/scripts/untyped-always-succeeds-txin.plutus"
+
+  -- This datum hash is the hash of the untyped 42
+  scriptDatumHash <- pure "9e1199a988ba72ffd6e9c269cadb3b53b5f360ff99f112d9b2ee30c4d74ad88b"
+  plutusRequiredSpace <- pure @_ @Integer 70000000
+  plutusRequiredTime <- pure @_ @Integer 70000000
+  datumFile <- H.note $ base </> "scripts/plutus/data/42.datum"
+  redeemerFile <- H.note $ base </> "scripts/plutus/data/42.redeemer"
+
+  -- Always succeeds Plutus script in use. Any datum and redeemer combination will succeed.
+  -- Script at: $plutusscriptinuse
+
+  -- Step 1: Create a tx ouput with a datum hash at the script address. In order for a tx ouput to be locked
+  -- by a plutus script, it must have a datahash. We also need collateral tx inputs so we split the utxo
+  -- in order to accomodate this.
+
+  plutusScriptAddr <- H.execCli
+    [ "address", "build"
+    , "--payment-script-file", plutusScriptFileInUse
+    , "--testnet-magic", show @Int testnetMagic
+    ]
+
+  -- mkdir -p $WORK
+
+  utxoAddr <- H.execCli
+    [ "address", "build"
+    , "--testnet-magic", show @Int testnetMagic
+    , "--payment-verification-key-file", utxoVKeyFile
+    ]
+
+  void $ H.execCli' execConfig
+    [ "query", "utxo"
+    , "--address", utxoAddr
+    , "--cardano-mode"
+    , "--testnet-magic", show @Int testnetMagic
+    , "--out-file", work </> "utxo-1.json"
+    ]
+
+  H.cat $ work </> "utxo-1.json"
+
+  utxo1Json <- H.leftFailM . H.readJsonFile $ work </> "utxo-1.json"
+  utxo1 <- H.noteShowM $ H.jsonErrorFail $ J.fromJSON @(HashMap Text Utxo) utxo1Json
+  txin <- H.noteShow $ head $ HM.keys utxo1
+  lovelaceAtTxin <- H.nothingFailM . H.noteShow $ utxo1 & HM.lookup txin <&> value >>= HM.lookup "lovelace"
+  lovelaceAtTxinDiv2 <- H.noteShow $ lovelaceAtTxin `div` 2
+
+  void $ H.execCli
+    [ "transaction", "build-raw"
+    , "--alonzo-era"
+    , "--fee", "0"
+    , "--tx-in", T.unpack txin
+    , "--tx-out", plutusScriptAddr <> "+" <> show @Integer lovelaceAtTxinDiv2
+    , "--tx-out-datum-hash", scriptDatumHash
+    , "--tx-out", utxoAddr <> "+" <> show @Integer lovelaceAtTxinDiv2
+    , "--out-file", work </> "create-datum-output.body"
+    ]
+
+  void $ H.execCli
+    [ "transaction", "sign"
+    , "--tx-body-file", work </> "create-datum-output.body"
+    , "--testnet-magic", show @Int testnetMagic
+    , "--signing-key-file", utxoSKeyFile
+    , "--out-file", work </> "create-datum-output.tx"
+    ]
+
+  void $ H.execCli' execConfig
+    [ "transaction", "submit"
+    , "--tx-file", work </> "create-datum-output.tx"
+    , "--testnet-magic", show @Int testnetMagic
+    ]
+
+  H.threadDelay 5000000
+
+  -- After "locking" the tx output at the script address, we can now can attempt to spend
```

Comment:
I would change this to something like: With the tx ouput at the script address we can now attempt to spend it

I need to update the comments in my bash script because technically the output isn't locked in this case.